src/guide/send_onchain.md

```diff
@@ -290,9 +296,10 @@ try {
 ```python
 destination_address = "bc1.."
 amount_sat = current_fees.min
+fee_hash = current_fees.fee_hash
 
 try:
-  sdk.send_onchain(amount_sat, destination_address, current_fees.fees_hash)
+  sdk_services.send_onchain(amount_sat=amount_msats, onchain_recipient_address="...", pair_hash=fee_hash)
```

Comment:
Is this missing the sat_per_vbyte parameter?

Reply:
Good catch, thanks! Updated.